Source/Core/Models/Event.cs

```diff
@@ -81,6 +81,7 @@ public static class KnownDataKeys {
             public const string SubmissionMethod = "@submission_method";
             public const string SessionEnd = "sessionend";
             public const string SessionHasError = "haserror";
+            public const string ManualStackingKey = "@manual_stacking_key";
```

Comment:
Sure, I can rename it. I would have to submit another pool request for the .NET client, since these names have to match.